src/loader.ts

```diff
@@ -81,6 +81,12 @@ async function ESBuildLoader(
 		} else {
 			/* Detect tsconfig file */
 
+			// Don't look for tsconfig.json based on external sources (see
+			// https://github.com/privatenumber/esbuild-loader/issues/363)
+			if (resourcePath.match(/node_modules/) !== null) {
```

Comment:
```suggestion
			if (resourcePath.includes('/node_modules/')) {
```

This could catch something like `node_modules_file`

Reply:
Great point, thanks for catching this! In the latest commit, I changed the search a little and hopefully the `path.sep` tactic means it is a little more portable